openquakeplatform/openquakeplatform/static/css/hrde.css

```diff
@@ -132,6 +132,26 @@ a {
   width: 100%;
 }
 
+.leaflet-control-layers {
+  width: 350px;
+}
+
+.leaflet-control {
+  clear: none;
+}
```

Comment:
Adding `overflow: hidden` you can avoid scroll bars in the control div:

From
![screenshot from 2015-05-15 15 33 19](https://cloud.githubusercontent.com/assets/1818657/7653693/01eb97cc-fb18-11e4-9140-affabf59fc7a.png)
To
![screenshot from 2015-05-15 15 32 58](https://cloud.githubusercontent.com/assets/1818657/7653696/05cef3b6-fb18-11e4-8a97-bcd3ac249014.png)